packages/mobile/src/storage/keychain.tsx

```diff
@@ -36,6 +46,9 @@ export async function retrieveStoredItem(key: string) {
     }
     return item.password
   } catch (error) {
+    if (isUserCancelledError(error)) {
+      throw error
+    }
```

Comment:
Maybe we should add a comment here why we're still re-throwing when cancelled?

Reply:
done! i hope it's clear from the comment..but i think it is still necessary/easier for the consumer of this function to be aware that fetching from the keychain failed (to know whether to log an error, to default to the pin input screen etc) - i toyed with the idea of returning null instead of throwing an error in this case, but then consumers need to know that null value means either the user cancelled or the keychain returned a falsey value :/ this seemed like the worse option

Reply:
i thought about this in the weekend, i don't think it should return an error....i will change this